src/layouts/DefaultLayout.tsx

```diff
@@ -14,9 +14,8 @@ export function DefaultLayout({ children }: Type) {
 
     return (
         <div
-            className={styles.motherContainer}
             data-theme={theme?.theme ? "dark" : "light"}
-            style={{ backgroundColor: theme?.theme ? "#262626" : "" }}
+            style={{ backgroundColor: theme?.theme ? "#262626" : "#f2fbf6", minHeight: "100vh" }}
```

Comment:
You've override my fix for the white space on larger screens

Reply:
the code is still the same the only difference is that I made the style inline it actually fixed the white spaces
should i still revert?